react/reducers/index.js

```diff
@@ -0,0 +1,83 @@
+import {
+  CHANNEL_UPDATE, CLAIM_UPDATE, ERROR_UPDATE, FILE_CLEAR, FILE_SELECTED, METADATA_UPDATE, PUBLISH_STATUS_UPDATE,
+  SET_PUBLISH_IN_CHANNEL,
+} from '../actions';
+
+const initialState = {
+  loggedInChannel: {
+    name   : null,
+    shortId: null,
+    longId : null,
+  },
+  publishInChannel: false,
+  status          : {
+    status : null,
+    message: null,
+  },
+  error: {
```

Comment:
Some of these should possibly be renamed or refactored into separate files (or possibly can wait until next refactor/revision).